From 6381b9485508560837f2ee43f06cb25d4709ae3e Mon Sep 17 00:00:00 2001
From: Kamil Debski <k.debski@samsung.com>
Date: Tue, 10 Sep 2013 18:08:49 +0200
Subject: [PATCH 0773/1302] ARM: dts: Correct USB PHY use in
 exynos4412-slp_pq.dts

Some values (ranges, address-cells, size-cells) were not used hence they
were removed.

Signed-off-by: Kamil Debski <k.debski@samsung.com>
Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>
---
 arch/arm/boot/dts/exynos4412-slp_pq.dts | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/arch/arm/boot/dts/exynos4412-slp_pq.dts b/arch/arm/boot/dts/exynos4412-slp_pq.dts
index 796c0c3..6907e59 100644
--- a/arch/arm/boot/dts/exynos4412-slp_pq.dts
+++ b/arch/arm/boot/dts/exynos4412-slp_pq.dts
@@ -1304,10 +1304,7 @@
 
 	exynos_usbphy: exynos-usbphy@125B0000 {
 		compatible = "samsung,exynos4212-usbphy";
-		reg = <0x125B0000 0x100 0x10020704 0x0c 0x1001021c 0x4>;
-		ranges;
-		#address-cells = <1>;
-		#size-cells = <1>;
+		reg = <0x125B0000 0x100>, <0x10020704 0x0c>, <0x1001021c 0x4>;
 		clocks = <&clock 305>, <&clock 2>, <&clock 2>, <&clock 2>,
 								<&clock 2>;
 		clock-names = "phy", "device", "host", "hsic0", "hsic1";
-- 
1.8.3.2

